benches/parallel_specs.rs

```diff
@@ -36,9 +36,9 @@ fn build() -> Planner<()> {
     {
         let ents: Vec<Entity> = w.create_iter().take(N).collect();
 
-        let mut rs = w.write::<RComp>();
-        let mut w1s = w.write::<W1Comp>();
-        let mut w2s = w.write::<W2Comp>();
+        let mut rs = w.write::<RComp>().pass();
```

Comment:
Combining the passes would be better here imo.
`let (mut rs, mut w1s, mut w2s) = (w.write::<RComp>(), w.write::<W1Comp>(), w.write::<W2Comp>()).pass()`